lib/pos_mempool.go

```diff
@@ -19,7 +19,7 @@ const (
 )
 
 type Mempool interface {
-	Start() error
+	Start(feeEstimatorPastBlocks []*MsgDeSoBlock, feeEstimatorNumBlocks uint64) error
```

Comment:
WDYT about having a separate Init(...) function, which would let you keep the Start() function scoped to just starting the mempool's event loop? Ideally, we're able to call Start() for a place where we don't have these params available on-hand. Decoupling the two now will open up some more options for us down the line
```suggestion
	Init(feeEstimatorPastBlocks []*MsgDeSoBlock, feeEstimatorNumBlocks uint64) error
	Start() error
```

Reply:
We can actually move the Init out of the interface I think!